fabfile.py

```diff
@@ -1,29 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
`fabfile.py` should be changed, not removed - it's the standard way, how we want to deploy this site.